src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        MyMap<Integer, String> map = new MyHashMap<>();
+        for (int i = 1; i < 13; i++) {
+            map.put(i,"line");
+        }
+        map.put(13, "some");
```

Comment:
Consider giving a more informative name to the value "line" and "some". Instead of a generic name, a name that reflects the context or the type of data being put in the map would enhance readability. For example, if these are messages, you could name them 'defaultMessage' and 'specialMessage', respectively.